framework/sentry/__init__.py

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env python3
 
 import logging
+from typing import Literal
 
-from sentry_sdk import init, capture_exception, capture_message, push_scope
+from sentry_sdk import init, capture_exception, capture_message, isolation_scope
```

Comment:
isolated scope creates new scope is current scope is not present, and if it is present, it builds upon it. So I thought that in case there is scope present, when we are capturing error, we would have more info in sentry and therefore it would be easier to debug